.github/ISSUE_TEMPLATE/5_performance_speed.md

```diff
@@ -46,6 +45,7 @@ assignees: ''
      so we know which functions might be causing it. See "How to Collect
      and Read Timeline Traces" on this blog post:
        https://medium.com/flutter/profiling-flutter-applications-using-the-timeline-a1a434964af3#a499
+     Make sure the performance overlay is turned OFF when recording the trace.
```

Comment:
Consider adding a brief explanation of how to do that -- i.e. press 'P' on the command line or the equivalent IDE action.

Reply:
Added a note.